### PR TITLE
feat: event table [CDF-6926]

### PR DESCRIPTION
**Pull request by polomani**
Screenshot 2021-02-18 at 10 42 05

### CODE REVIEW

**Review by dbrattli — APPROVED** [review]
Nice work!

File: src/components/queryHelp.tsx [hunk 1]
```diff
@@ -1,99 +1,163 @@
...
+    <br />
+    <Code>{`ts{} + sin(ts{granularity='24h', aggregate='average'})`}</Code>
+    <br />
+    <br />
+    Variable length functions (sum, max, min, avg) can also be applied to all filtered timeseries.
+    Examples:
+    <br />
+    <Code>{`sum(ts{metadata={type="TEMP"}})`}</Code>
+    <br />
+    ↪ yields one timeseries that is thesum of all temperature timeseries
```
Comment (tengelskar) [inline src/components/queryHelp.tsx:81]:
```suggestion
    ↪ yields one timeseries that is the sum of all temperature timeseries
```

File: src/components/queryHelp.tsx [hunk 2]
```diff
@@ -1,99 +1,163 @@
...
+    <Code>!~</Code> – regex inequality, excludes results satisfying the regular expression
+    <br />
+    <Code>!=</Code> – strict inequality, returns items where property doesn&apos;t equal a given
+    value
+    <br />
+    <br />
+    Example: <Code>{`events{type='WORKORDER', subtype=~'SUB.*'}`}</Code>
+    <br />
+    <span className="gf-formatted-warning">
+      Note: client-side filters should not be used as a primary filtering method.
```
Comment (arnekv) [inline src/components/queryHelp.tsx:153]:
```suggestion
      Note: Do not use client-side filters as the primary filtering method.
```

File: src/components/queryHelp.tsx [hunk 3]
```diff
@@ -1,99 +1,163 @@
...
+      <br />
+      Those filters are applied after items has been returned from CDF, therefore there is a risk of
```
Comment (arnekv) [inline src/components/queryHelp.tsx:155]:
```suggestion
      The filters are applied after items have been returned from CDF, and there is a risk that
```

File: src/components/queryHelp.tsx [hunk 4]
```diff
@@ -1,99 +1,163 @@
...
+      not seeing all data if CDF returned the maximum number of items (1000).
```
Comment (arnekv) [inline src/components/queryHelp.tsx:156]:
```suggestion
      you will not see all data if CDF returned the maximum number of items (1000).
```

File: src/components/queryHelp.tsx [hunk 5]
```diff
@@ -1,99 +1,163 @@
...
+    </span>
+    <br />
+    Templating is also available by using <Code>$variable_name</Code> syntax.
```
Comment (arnekv) [inline src/components/queryHelp.tsx:159]:
```suggestion
    Templating is available by using the <Code>$variable_name</Code> syntax.
```

**Review by arnekv — APPROVED** [review]
I've left a few suggestions - please check that I haven't misunderstood/changed the technical meaning.

**Pull request merged**